Dockerfile

```diff
@@ -1,12 +1,19 @@
+FROM alpine:3.18
+
+WORKDIR /root
+
+RUN mkdir -p /usr/share/GeoIP
+COPY ./GeoLite2-City.mmdb /usr/share/GeoIP/GeoLite2-City.mmdb
+COPY ./GeoLite2-Country.mmdb /usr/share/GeoIP/GeoLite2-Country.mmdb
+
 # Dockerfile - alpine
 # https://github.com/openresty/docker-openresty
 
 ARG RESTY_IMAGE_BASE="alpine"
```

Comment:
If you are defining these ARGs, you should use them, like in the original code:
`FROM ${RESTY_IMAGE_BASE}:${RESTY_IMAGE_TAG}`

Reply:
I made this change because our docker build expects the FROM at the fist line. So, I have used the FROM alpine:3.18. The error message we have if we do not have FROM on first line is 
Error response from daemon: no build stage in current context
exit status 1
failure build reference is given below
https://drone-gh.acp.homeoffice.gov.uk/UKHomeOffice/dq-nginx-proxy-redirect/151/1/4

Reply:
w.r.t openssl 1.1.1f we are not moving on docker container. The scope of openssl migration comment is only on ec2 instance as per ITHC. We do not want to introduce too many changes at first build. 
  When there is a requirement for that we will take it at that stage.